Hymns/Tags/TagSheetView.swift

```diff
@@ -66,8 +68,8 @@ struct TagSheetView: View {
                     }, label: {
                         HStack {
                             Text("Create a new tag").font(.headline).fontWeight(.bold)
-                            Image(systemName: self.viewModel.showNewTagCreation ? "chevron.down" : "chevron.up")
-                        }.foregroundColor(.primary).padding(.vertical)
+                            Image(systemName: self.viewModel.showNewTagCreation ? "chevron.up" : "square.and.pencil")
+                        }.foregroundColor(self.viewModel.showNewTagCreation ? .primary : .accentColor).padding(.vertical)
```

Comment:
In the case where the user is going to need to click the button "create a new tag" to bring in the drop down. I think it is good to make it more clear we have a button here with functionality. The accent color in this case makes this much more apparent. I did change to square.and.pencil as well which I think is a much more clear button. However, you all are free to disagree and override this change.